.github/workflows/codeql.yml

```diff
@@ -29,7 +29,6 @@ jobs:
       - name: Initialize CodeQL
         uses: github/codeql-action/init@v3
         with:
-          config-file: ./.github/codeql/codeql-config.yml
```

Comment:
Why remove the config?

Reply:
Because it doesn't exist since it's not needed.

Reply:
isn't it this one? https://github.com/Havunen/detective-vue2/blob/main/.github/workflows/codeql.yml

Reply:
ah config, right, LGTM